nexteuropa_theme_functions/nexteuropa_theme_functions.helpers.inc

```diff
@@ -193,8 +193,8 @@ function _nexteuropa_theme_functions_add_datepicker(&$element) {
 /**
  * Helper function to retrieve the text fields.
  */
-function _nexteuropa_theme_functions_textfields() {
-  $fields = field_read_fields(['type' => 'text_long']);
+function _nexteuropa_theme_functions_textfields($type = 'text_long') {
```

Comment:
This change is unrelated, but doing like this we would have the chance to retrieve a list of fields basing on its type which was something i thought i was needing.